### PR TITLE
[@types/raygun4js] Export internal interfaces

### PR DESCRIPTION
**Pull request by krishnaKapadia**
This PR exports some of the internal interfaces that are not being exported currently.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: www.raygun.com.
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Review by BenjaminHarding — APPROVED** [review]
Looks good. Thanks for the updates.

If you have a bit more time there are a few more additions that could be done! 

1. Add the `captureMissingRequests`, `clientIp`, and `automaticPerformanceCustomTimings` to the `RaygunOptions`, see [the initialisation options](https://github.com/MindscapeHQ/raygun4js#initialization-options).
2. Add the `endSession` method on the `RaygunV2` interface, see how you can [end a session](https://github.com/MindscapeHQ/raygun4js#ending-a-session)

**Comment by krishnaKapadia** [comment]
> Looks good. Thanks for the updates.
> 
> If you have a bit more time there are a few more additions that could be done!
> 
> 1. Add the `captureMissingRequests`, `clientIp`, and `automaticPerformanceCustomTimings` to the `RaygunOptions`, see [the initialisation options](https://github.com/MindscapeHQ/raygun4js#initialization-options).
> 2. Add the `endSession` method on the `RaygunV2` interface, see how you can [end a session](https://github.com/MindscapeHQ/raygun4js#ending-a-session)

Have added these updated to this PR. Please re-review @BenjaminHarding

File: types/raygun4js/index.d.ts [hunk 1]
```diff
@@ -115,6 +115,22 @@ interface RaygunOptions {
      * String which can be optionally set "onLoad" which will then boot the RealUserMonitoring side instead of waiting for the `load` event.
      */
     from?: string | "onLoad";
+
+    /**
+     * Raygun4JS will track each window.performance.measure call as a custom timing entry. This enables developers to use a more native API for tracking performance timings.
+     */
+    automaticPerformanceCustomTimings ?: boolean;
```
Comment (BenjaminHarding) [inline types/raygun4js/index.d.ts:122]: Excessive space between `Timings` and `?`

**Review by BenjaminHarding — APPROVED** [review]

**Review by krishnaKapadia — COMMENTED** [review]
LG2M 🚀

**Review by BenjaminHarding — APPROVED** [review]

**Comment by krishnaKapadia** [comment]
Ready to merge

**Pull request merged**